test/resource_marketplace.test.js

```diff
@@ -184,7 +184,7 @@ describe('Marketplace Resource', function() {
       expect(notifications[0].type).to.equal('seller_listing_purchased')
       expect(notifications[0].status).to.equal('unread')
 
-      await marketplace.acceptOffer('999-001-0-0')
+      await marketplace.acceptOffer('999-001-0-0', {})
```

Comment:
I think we should use default parameters so that we don't have to pass in empty objects. See other comment.

Reply:
Sounds good. Done.